src/app/utils.ts

```diff
@@ -6,6 +6,8 @@ export const GRAPH_REFRESH_INTERVAL = 5000; // milliseconds
 export const STATS_HISTORY_TIME_FILTER = '10'; // minutes
 export const ASSET_READINGS_TIME_FILTER = 600; // seconds
 
+export const SERVICE_TYPES_LIST = ["Core", "Storage", "Southbound", "Northbound", "Notifications", "Management"];
```

Comment:
SUPPORTED_SERVICE_TYPES
add Dispatcher too, and please filter that while using ACL page.